packages/atomic/src/components/result-template-components-v1/atomic-result-icon/assets/attachment.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" aria-label="attachment" enable-background="new 0 0 60 60" focusable="false" height="60" role="img" viewBox="0 0 60 60" width="60"><title>attachment</title><path clip-rule="evenodd" d="m3.753 60c-2.073 0-3.753-1.667-3.753-3.753v-52.494c0-2.073 1.667-3.753 3.753-3.753h52.494c2.073 0 3.753 1.667 3.753 3.753v52.493c0 2.074-1.667 3.754-3.753 3.754z" fill="#00a78a" fill-rule="evenodd"></path><path d="m24.957 36.432c.362.362.906.302 1.268 0l6.039-6.039c.423-.423 1.147-.483 1.691 0s .483 1.329 0 1.812l-7.428 7.307c-1.631 1.631-4.348 1.631-5.979 0l-.06-.06c-1.631-1.631-1.631-4.348 0-5.979l13.105-13.105c1.631-1.631 4.348-1.631 5.979 0l .06.06c1.631 1.631 1.631 4.348 0 5.979l-.06.06c-.302.302-.362.725-.121 1.087.362.664.664 1.389.845 2.114.121.483.664.604 1.027.302.483-.483.906-.966.906-.966 3.08-3.08 2.959-8.093-.121-11.173l0 0c-3.08-3.08-8.093-3.08-11.173 0l-13.104 13.045c-3.08 3.08-3.08 8.093 0 11.173l.121.121c3.08 3.08 8.032 3.08 11.112 0l7.489-7.428c1.933-1.933 1.872-5.073-.121-7.005-1.933-1.872-5.073-1.751-6.945.181l-5.918 5.918c-.362.362-.362.966 0 1.329z" fill="#fff"></path></svg>
```

Comment:
Why do the svg icons need to be copied to a `v1` directory? Wouldn't they be the same between v0 and v1?

Reply:
Yes, I copied the whole folder. I'll remove them from this PR and change the path